TFEngine.py

```diff
@@ -59,7 +59,8 @@ def __init__(self, engine, dataset, batches, train, eval=True, train_flag=None,
     :param bool train: whether to do updates on the model
     :param bool|None train_flag: normally just as train. but e.g. maybe you want to have the train_flag but not train
     :param bool eval: whether to evaluate (i.e. calculate loss/error)
-    :param dict[str,tf.Tensor|TFUtil.Data|TFNetworkLayer.LayerBase]|None extra_fetches: additional fetches per step.
+    :param dict[str,tf.Tensor|TFUtil.Data|TFNetworkLayer.LayerBase|lambda]|None extra_fetches:
```

Comment:
Note: `lambda` is not a valid type. Use `()->tf.Tensor`.

Reply:
Fixed.